cfgov/cfgov/settings/base.py

```diff
@@ -37,8 +39,17 @@
 PASSWORD_HASHERS = global_settings.PASSWORD_HASHERS
 
 # Application definition
-INSTALLED_APPS = (
-    "permissions_viewer",
+if wagtail.VERSION < (2, 10):
+    INSTALLED_APPS = (
+        "permissions_viewer",
+    )
+else:
+    INSTALLED_APPS = (
+        "permissions_viewer",
+        "wagtail.contrib.legacy.richtext",
+    )
```

Comment:
This could all be simplified:

```suggestion
INSTALLED_APPS = (
    "permissions_viewer",
)
if wagtail.VERSION >= (2, 10):
    INSTALLED_APPS += (
        "wagtail.contrib.legacy.richtext",
    )
```

Reply:
I will use your recommended simplification